pkg/reconciler/apis/apibinding/apibinding_controller.go

```diff
@@ -67,6 +67,8 @@ func NewController(
 	apiBindingInformer apisinformers.APIBindingInformer,
 	apiExportInformer apisinformers.APIExportInformer,
 	apiResourceSchemaInformer apisinformers.APIResourceSchemaInformer,
+	temporaryRootShardApiExportInformer apisinformers.APIExportInformer, /*TODO(p0lyn0mial): replace with multi-shard informers*/
+	temporaryRootShardApiResourceSchemaInformer apisinformers.APIResourceSchemaInformer, /*TODO(p0lyn0mial): replace with multi-shard informers*/
```

Comment:
in all this wiring. Shouldn't we go with `temporaryOtherShard....Informer`? The same code will run on the root shard, and it must watch the second shard.

Reply:
we could rename to `temporaryLocalShard...Informer`, wdyt?